sdk/python/.gitignore

```diff
@@ -6,3 +6,4 @@
 .venv/
 venv/
 .coverage
+build/
```

Comment:
Why's this changed?

Reply:
For some reason we weren't ignoring build artifacts in this SDK before - I can remove it but it makes development one step more difficult.

Reply:
oh that's fine, was just wondering.